com.unity.render-pipelines.universal/Runtime/2D/Passes/Render2DLightingPass.cs

```diff
@@ -114,8 +138,10 @@ private short GetCameraSortingLayerBoundsIndex()
             ref DrawingSettings drawSettings,
             ref RenderTextureDescriptor desc)
         {
+            bool drawLights = (DebugHandler == null) || DebugHandler.IsLightingActive;
```

Comment:
Note @unity-cchu ...

It seems this flag only disables the volumetric... but does it mean that we disable the entire lighting system?
If so we have to do more (don't render any lights into RT and make RT white) or set the shader keywords to indicate no light textures are valid.

Reply:
it seems that this is to fix the fact that volumetric does not turn off when we "Disable Lighting" from sceneview... is that correct?

Reply:
Yes - "IsLightingActive" is there to say that the user requires to see the view without any lighting at all, so all forms of lighting should be disabled when this is "true"

Reply:
@UnityPaul Where do we need to make these other changes to ensure we're disabling all lighting?